scripts/insert-includes.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
This breaks the flow for non-conda users :(

Perhaps we should just leave this script be, and change the callsite of the script to `python insert-includes.py`, rather than `./insert-includes.py`, and let the environment decide which python to use?

Reply:
Wouldn't `/usr/bin/env` handle cases for non-conda users as well? This is what other python scripts in `scripts/` like `numaprefix` and `uniquify-module-names.py` are already doing.

Reply:
Huh you are right, I don't know how this works then. This looks good in that case.